content/gherkin.md

```diff
@@ -44,14 +45,16 @@ Feature: Some terse yet descriptive text of what is desired
     # ...
 ```
 
-The `Feature` starts on the first line. Lines 2–4 are unparsed (free format) text, which is expected to
-describe the business value of this feature. The keyword `Scenario` on line 6 starts a scenario.
-Lines 7–13 are the steps for the scenario. Line 15 starts the next Scenario, and so on.
+The feature starts on the first line with the keyword `Feature`. The lines below are unparsed (i.e. free format) text, which you can use to
+describe this feature. The keyword `Scenario` starts a scenario, followed by the steps for the scenario. The next scenario is started by the keyword `Scenario` again.
+The text after the keyword `Scenario` -but still on the same line- is also free format text to describe your scenario.
 
 ## Keywords
 
-In Gherkin, each line that isn't blank (except for the feature and scenario descriptions) has to start with a Gherkin *keyword*,
-followed by any text you like. The main keywords are:
+Except for the free format feature and scenario descriptions, each line that isn't blank has to start with a Gherkin *keyword*,
```

Comment:
`free format feature ` -> `feature description` perhaps?

Reply:
Updated to: "Each line that isn't a blank line has to start with a Gherkin *keyword*, followed by any text you like. The only exceptions are the feature and scenario descriptions." Let me know what you think.